recipe/provision.php

```diff
@@ -100,7 +100,7 @@
 
 desc('Upgrades all packages');
 task('provision:upgrade', function () {
-    run('apt-get upgrade -y', ['env' => ['DEBIAN_FRONTEND' => 'noninteractive']]);
+    run('apt-get upgrade -y', ['env' => ['DEBIAN_FRONTEND' => 'noninteractive']], 900);
```

Comment:
Let’s better to put it in second arg as timeout => 900

Reply:
Done! :wink: